ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__periodics.yaml

```diff
@@ -204,6 +204,7 @@ tests:
     cluster_profile: vsphere
     test:
     - ref: openshift-extended-test
+    - ref: openshift-extended-web-tests
```

Comment:
@yapei vsphere test is already updated too.

Reply:
@kuiwang02 thanks for pointing out : ) 
I just use it for debugging